tests/codeception/acceptance/SubThemeCest.php

```diff
@@ -90,6 +90,24 @@ public function testSubTheme(AcceptanceTester $I) {
     $I->canSeeResponseCodeIs(200);
   }
 
+/**
+   * Enable the minimally branded subtheme and the config should reflect the changes done.
+   * Test the changes are there.
+   * @group minimal-subtheme-test
+   */
+  public function testMinimalSubtheme(AcceptanceTester $I) {
+    $I->runDrush('theme:enable -y minimally_branded_subtheme');
+    $I->logInWithRole('administrator');
+    $I->amOnPage('/admin/appearance');
+    $I->click('Set as default', 'a[title="Set Stanford Minimally Branded Subtheme as default theme"]');
+    $I->amOnPage('/');
+    $I->canSeeResponseCodeIs(200);
+    $I->dontSee('.su-lockup__wordmark');
+    $I->dontSee('.su-brand-bar__logo');
+    $I->dontSee('.su-global-footer__container');
+    $I->dontSee('.su-brand-bar--default');   
```

Comment:
These pass because the `dontSee` is actually looks for text. If you want to check for a class, you'll want to use `$I->dontSeeElement`

Reply:
Ok. Lookin'